test/e2e/storage/persistent_volumes-disruptive.go

```diff
@@ -159,15 +159,17 @@ var _ = framework.KubeDescribe("PersistentVolumes [Volume][Disruptive][Flaky]",
 func testKubeletRestartsAndRestoresMount(c clientset.Interface, f *framework.Framework, clientPod *v1.Pod, pvc *v1.PersistentVolumeClaim, pv *v1.PersistentVolume) {
 	By("Writing to the volume.")
 	file := "/mnt/_SUCCESS"
-	_, err := podExec(clientPod, fmt.Sprintf("touch %s", file))
+	out, err := podExec(clientPod, "touch "+file)
```

Comment:
I've also seen both used. I went with concatenation for slightly better readability since it's such a short string and only inserts 1 var.